doc/user/content/sql/create-role.md

```diff
@@ -22,20 +22,35 @@ _role_name_         | A name for the role.
 
 ## Details
 
-Unlike PostgreSQL, Materialize derives the `LOGIN` and `SUPERUSER`
-attributes for a role during authentication, every time that role tries
-to connect. Therefore, you cannot specify either
-attribute when creating a new role. Additionally, we do not support the
-`CREATE USER` command, because it implies a `LOGIN` attribute for the role.
+Materialize's support for `CREATE ROLE` is similar to that of PostgreSQL, with
+the following options exceptions:
 
-Unlike PostgreSQL, Materialize does not currently support `NOINHERIT`.
+| Option   | Description                                                                                                                                                                                            |
+|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `INHERIT`   | Materialize implicitly uses `INHERIT` for the `CREATE ROLE` command. That is, `CREATE ROLE <name>` and `CREATE ROLE <name> WITH INHERIT` are equivalent.                                                   |
+| `NOINHERIT` | Materialize does not support the `NOINHERIT` option for `CREATE ROLE`.                                                                                                                                  |
+| `LOGIN`     | Materialize does not support the `LOGIN` option for `CREATE ROLE`.<ul><li>Instead, Materialize derives the `LOGIN` option for a role during authentication every time that role tries to connect.</li><li>Materialize does not support the `CREATE USER` command as the command implies a `LOGIN` attribute for the role.</li></ul>|
```

Comment:
Nit, to take or leave: I'm not sure you need bullet points here! It might look simpler to just have a couple of sentences. But totally up to you!